PayPal/src/main/java/com/braintreepayments/api/PayPalLineItem.java

```diff
@@ -178,6 +227,21 @@ public String getUrl() {
         return url;
     }
 
+    @Nullable
```

Comment:
```suggestion
    @NonNull
```
The getter and setter annotations need to match. Not sure why, but I remember running into this issue a while back.

Reply:
These properties are optional fields. So, we can leave it as @Nullable. Let me know if you think otherwise.